test/integration/force-gzip.js

```diff
@@ -52,7 +52,7 @@ describe('handle forced gzip', function () {
       host: '127.0.0.1',
       port: config.www_port,
       method: 'get',
-      path: '/_api/?gzip=force',
+      path: '/_api/?force_gzip=true',
```

Comment:
Would it make sense to add this here?

```
headers: {
  'Accept-Encoding': 'deflate'
}
```

Just to make clear that the request did not request gzip in its headers?